techlibs/xilinx/synth_xilinx.cc

```diff
@@ -84,11 +84,8 @@ struct SynthXilinxPass : public ScriptPass
 		log("    -nodsp\n");
 		log("        do not use DSP48E1s to implement multipliers and associated logic\n");
 		log("\n");
-		log("    -iopad\n");
-		log("        enable I/O buffer insertion (selected automatically by -ise)\n");
-		log("\n");
 		log("    -noiopad\n");
-		log("        disable I/O buffer insertion (only useful with -ise)\n");
+		log("        disable I/O buffer insertion\n");
```

Comment:
Perhaps add the reason why you would want this: `(useful for hierarchical or out-of-context flows)

Reply:
addressed in 436fea9e6990c66369d7c30b571920ae115efb44